lib/strategy.js

```diff
@@ -14,6 +27,19 @@ function getHost(region) {
   }
 }
 
+/**
+ * `getMasheryHost` utility function
+ *
+ * Takes a two character region code and returns a Mashery host URL
+ *
+ *Examples:
+ *
+ *     getMasheryHost('us');  // returns 'us.api.battle.net'
+ *
+ * @param {String} region
+ * @return {String}
+ * @api protected
```

Comment:
What would be a better label for this? `getMasheryHost` is not exported from this module, so I don't think @public applies.